### PR TITLE
Show external ID and linkt to external CCS on shadow difference page

### PR DESCRIPTION
**Pull request by nickygerritsen**
image

The comment thing we need to think about how to do this and where to store it so I only did the first part of the issue now.

### CODE REVIEW

**Comment by nickygerritsen** [comment]
I'm not sure yet if showing "External" twice is very useful. Better UX is welcome.

File: webapp/templates/jury/partials/submission_list.html.twig [hunk 1]
```diff
@@ -191,13 +202,28 @@
                 {% else %}
                     {% set externalJudgement = submission.externalJudgements.first %}
                 {% endif %}
+                {% if external_ccs_submission_url is not empty %}
+                    {% set externalSubmissionUrl = submission | externalCcsUrl %}
+                {% else %}
+                    {% set sexternalSubmissionUrl = null %}
```
Comment (meisterT) [inline webapp/templates/jury/partials/submission_list.html.twig:208]: Typo: set **s**external

File: webapp/templates/jury/partials/submission_list.html.twig [hunk 2]
```diff
@@ -191,13 +202,28 @@
...
                 <td>
-                    <a href="{{ link }}">
+                    <a {% if externalSubmissionUrl is not null %}href="{{ externalSubmissionUrl }}"{% endif %}>
                         External
                     </a>
                 </td>
                 <td>
-                    <a href="{{ link }}">
+                    <a {% if externalSubmissionUrl is not null %}href="{{ externalSubmissionUrl }}"{% endif %}>
+                        s{{ submission.externalid }}
```
Comment (meisterT) [inline webapp/templates/jury/partials/submission_list.html.twig:217]: remove s - it adds more confusion, if external IDs are not numeric but alphanumeric

File: webapp/templates/jury/partials/submission_list.html.twig (hunk 1 above)
Comment (nickygerritsen) [reply webapp/templates/jury/partials/submission_list.html.twig:208]: I lol'ed

**Pull request merged**